AutoGraph/Dispatcher.swift

```diff
@@ -78,6 +78,14 @@ open class Dispatcher {
             if let variables = try sendable.variables?.graphQLVariablesDictionary() {
                 parameters["variables"] = variables
             }
+
+            if let operation = sendable.queryDocument as? Operation {
+                parameters["operationName"] = operation.name
+            }
+            else if let document = sendable.queryDocument as? Document, let operation = document.operations.first {
+                parameters["operationName"] = operation.name
+            }
```

Comment:
note: at some point we should lift documents into an api that allows selecting which operation, but we've never supported that in the past so this is fine for now.